internal/toolkit/toolkit.go

```diff
@@ -145,6 +146,17 @@ func (d *DefaultToolkit) GetInput(name string) (string, bool) {
 	return s, ok
 }
 
+func (d *DefaultToolkit) GetInputList(name []string) ([]string, bool) {
+	d.once.Do(d.init)
+	var ok bool
+	for _, str := range name {
+		s, ok := d.Environment[fmt.Sprintf("INPUT_%s", strings.ToUpper(str))]
```

Comment:
I think instead of iterating over a list of `name`s, you'll need to parse the list from a single env var. We may want to define the `blocked-namespaces` as a comma separated list.

Reply:
Hi @jkutner , have tried to fix it, please feel free to have a look, thanks